.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+
+name: build
+
+on: push
+
+jobs:
+
+  test:
+
+    name: Test on Ruby ${{ matrix.ruby }}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby:
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby ${{ matrix.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+
+      - name: Test
+        run: bundle exec rake --trace spec rubocop
```

Comment:
🎨 If not using the default `rake` task, I'd suggest adding a task that depends on whatever runs in CI, so any changes to CI tasks can be managed from the `Rakefile` alone.

Reply:
Good thinking. In 18737c5 I've changed it to use the default Rake task.